package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
```

Comment:
Please add the standard header – copy from the top of rms.py

Reply:
Please add `from __future__ import absolute_import` as the very first import with any import from `six.moves` directly below it. Also include the `range` import from six.moves